test/e2e/tooltip.test.js

```diff
@@ -81,4 +81,25 @@ describe('vl-tooltip', async () => {
         const tooltip = await vlTooltipPage.getStaticTooltip();
         await assertTooltipIsZichtbaarMetText(tooltip, 'Static Tooltip');
     });
+
+    it('als gebruiker krijg ik alleen de tooltip te zien wanneer ik over een element hover met tooltip', async () => {
+        const button = await vlTooltipPage.getTopTooltipButton();
+        const buttonInsideShadowDOM = await vlTooltipPage.getShadowDOMTooltipButton();
+        const tooltip = await vlTooltipPage.getTopTooltip();
+        const tooltipInsideShadowDOM = await vlTooltipPage.getShadowDOMTooltip();
+        await assert.eventually.isFalse(tooltip.isDisplayed());
+        await assert.eventually.isFalse(tooltipInsideShadowDOM.isDisplayed());
+        await button.hover();
+        await assert.eventually.isTrue(tooltip.isDisplayed());
+        await assert.eventually.isFalse(tooltipInsideShadowDOM.isDisplayed());
+        await buttonInsideShadowDOM.hover();
+        await assert.eventually.isFalse(tooltip.isDisplayed());
+        await assert.eventually.isTrue(tooltipInsideShadowDOM.isDisplayed());
+        await button.hover();
+        await assert.eventually.isTrue(tooltip.isDisplayed());
+        await assert.eventually.isFalse(tooltipInsideShadowDOM.isDisplayed());
+        await buttonInsideShadowDOM.hover();
+        await assert.eventually.isFalse(tooltip.isDisplayed());
+        await assert.eventually.isTrue(tooltipInsideShadowDOM.isDisplayed());
```

Comment:
waarom 2 keer?

Reply:
De bug was dat de tooltip niet meer verdween na een blur. Door 2 keer te hoveren veroorzaak ik ook 2 keer een blur en ben ik zeker dat de bug opgelost is voor elementen met een shadow DOM parent en de bug zich ook niet voordoet voor de andere elementen.